sql/core/src/test/resources/sql-tests/results/except-all.sql.out

```diff
@@ -230,10 +230,9 @@ org.apache.spark.sql.AnalysisException
 {
   "errorClass" : "NUM_COLUMNS_MISMATCH",
   "messageParameters" : {
-    "invalidNumColumns" : "2",
-    "invalidOrdinalNum" : "second",
+    "leftNumCols" : "1",
     "operator" : "EXCEPTALL",
```

Comment:
hmm, could you output it as `EXCEPT ALL`

Reply:
Done